pkg/controller/disruption/disruption.go

```diff
@@ -541,6 +588,35 @@ Pod:
 	return
 }
 
+// Builds new PodDisruption map, possibly removing items that refer to non-existing, already deleted
+// or not-deleted at all items. Also returns an information when this check should be repeated.
+func buildDisruptedPodMap(pods []*api.Pod, pdb *policy.PodDisruptionBudget, currentTime time.Time) (map[string]unversioned.Time, *time.Time) {
+	disruptedPods := pdb.Status.DisruptedPods
+	result := make(map[string]unversioned.Time)
+	var recheckTime *time.Time
+
+	if disruptedPods == nil || len(disruptedPods) == 0 {
+		return result, recheckTime
+	}
+	for _, pod := range pods {
+		if pod.DeletionTimestamp == nil {
```

Comment:
Can you make this 
```go
if pod.DeletionTimestamp != nil {
   continue
}
disruptionTime, found := disruptedPods[pod.Name]
if !found {
   continue
}
```
Will deindent the rest of the code and make it more readable imo.